packages/orchestration/test/examples/sendAnywhere.test.ts

```diff
@@ -0,0 +1,153 @@
+  ibcHooksEnabled: false,
+  icaEnabled: false,
+  icqEnabled: false,
+  pfmEnabled: false,
+};
+
+const connectionDefaults = {
+  versions: [
+    {
+      identifier: '1',
+      features: ['ORDER_ORDERED', 'ORDER_UNORDERED'],
+    },
+  ],
+  delay_period: 0n,
+};
+
+const txChannelDefaults = {
+  counterPartyPortId: 'transfer',
+  version: 'ics20-1',
+  portId: 'transfer',
+  ordering: 1, // ORDER_UNORDERED
+  state: 3, // STATE_OPEN
+};
+
+test('send using arbitrary chain info', async t => {
+  t.log('bootstrap, orchestration core-eval');
+  const {
+    bootstrap,
+    commonPrivateArgs,
+    brands: { ist },
+    utils: { inspectLocalBridge },
+  } = await commonSetup(t);
+
+  const { rootZone: zone, board, marshaller } = bootstrap;
+  const timerBrand = await E(bootstrap.timer).getTimerBrand();
+
+  const { zoe, bundleAndInstall } = await setUpZoeForTest();
+
+  t.log('contract coreEval', contractName);
+
+  const installation: Installation<StartFn> =
+    await bundleAndInstall(contractFile);
+
+  const { instance, creatorFacet } = await E(zoe).startInstance(
+    installation,
+    { Stable: ist.issuer },
+    {},
+    {
+      timerBrand,
+      board,
+      marshaller,
+      ...commonPrivateArgs,
+      storageNode: await E(bootstrap.storage.rootNode).makeChildNode(
+        contractName,
+      ),
+    },
+  );
+
+  t.log('admin adds hot new chain to contract');
+  const hotChainInfo = harden({
+    chainId: 'hot-new-chain-0',
+    stakingTokens: [{ denom: 'uhot' }],
+    ...chainInfoDefaults,
+  }) as CosmosChainInfo;
+  const agoricToHotConnection = {
+    ...connectionDefaults,
+    id: 'connection-1',
+    client_id: '07-tendermint-1',
+    state: 3, // STATE_OPEN
+    counterparty: {
+      client_id: '07-tendermint-2109',
+      connection_id: 'connection-1649',
+      prefix: {
+        key_prefix: 'aWJj',
+      },
+    },
+    transferChannel: {
+      counterPartyChannelId: 'channel-1',
+      channelId: 'channel-0',
+      ...txChannelDefaults,
+    },
+  } as IBCConnectionInfo;
+  t.log('add chain using creatorFacet', hotChainInfo.chainId);
+  const chainKey = await E(creatorFacet).addChain(
+    hotChainInfo,
+    agoricToHotConnection,
+  );
+
+  t.log('client uses contract to send to hot new chain');
+  const publicFacet = await E(zoe).getPublicFacet(instance);
+  const inv = E(publicFacet).makeSendInvitation();
+
+  const amt = await E(zoe).getInvitationDetails(inv);
+  t.is(amt.description, 'send');
+
+  const anAmt = ist.make(20n);
+  const Send = ist.mint.mintPayment(anAmt);
+  const dest = { destAddr: 'hot1destAddr', chainKey };
+  const userSeat = await E(zoe).offer(
+    inv,
+    { give: { Send: anAmt } },
+    { Send },
+    dest,
+  );
+  const result = await E(userSeat).getOfferResult();
+  t.is(result, undefined);
+
+  const history = inspectLocalBridge();
+  t.like(history, [
+    {
+      type: 'VLOCALCHAIN_ALLOCATE_ADDRESS',
+    },
+    {
+      address: 'agoric1fakeLCAAddress',
```

Comment:
this isn't part of the spec, just incidental.
```suggestion
```

down below too for `sender`. or if you want to test that they match, read the `address` from this object and reference it in `sender`